services/github/auth/acceptor.spec.js

```diff
@@ -43,7 +43,10 @@ describe('Github token acceptor', function() {
   })
 
   beforeEach(function() {
-    acceptor.setRoutes(camp)
+    acceptor.setRoutes({
+      server: camp,
+      onTokenAccepted: () => {},
```

Comment:
If I'm following the code flow correctly, this is what was extracted from the persistence implementations with the responsibility shifting to the caller. If that's right then should we attach a spy to this function to make sure it is called with the expected value?

Reply:
Yes, that makes sense. There was not yet a test of this part of the flow so I added one.